File: OMSWeb/Models/Tracks/Station.cs
namespace OMSWeb.Models.Tracks
{
  public class Station
  {
    public int Id { get; set; }
    public string PhysicalId { get; set; }
    public string LogicalId { get; set; }
    public int? PointId { get; set; }
    public string Direction { get; set; }
    public int? CarrierType { get; set; }
    public int? NextPoint { get; set; }
    public int? Offset { get; set; }
    public bool? Unuse { get; set; }
    public string CarrierId { get; set; }
    public string User { get; set; }
    public string Note { get; set; }
    
    public string? CAlias { get; set; }
  }
}

File: OMSWeb/frontend/packages/oms-track-monitor/src/TrackObjects/station/types/Station.ts
import { CanBeCarrierFocused } from 'src/MapObjects/focus/types/CanBeCarrierFocused'
import { CanBeFocused } from 'src/MapObjects/focus/types/CanBeFocused'

type Station = {
  id: number
  logicalId: string
  physicalId: string

  direction: string
  pointId: number
  nextPoint: number
  offset: number

  unuse?: boolean
  state?: number

  group?: number
  carrierType?: string
  carrierId? : string

  user?: string
  note?: string

  cAlias?: string
} & CanBeFocused &
  CanBeCarrierFocused

export { Station }
